dagshub/streaming/mount.py

```diff
@@ -89,6 +123,17 @@ def getattr(self, path, fd=None):
             raise FuseOSError(errno.ENOENT)
 
     def read(self, path, size, offset, fh):
+        """
+        Read data from a file.
```

Comment:
I think saying that this reads data in the form of bytes is important (if that is the case)